context.annotation.value.empty=Annotation "org.kaleidofoundry.core.context.Context" value property is empty
context.annotation.field.illegal=Given field have to be annotated with org.kaleidofoundry.core.context.annotation.{0}

context.annotation.illegaluse.noRuntimeContextField=The field "{0}" is annotated by "{1}", but its class does not contains RuntimeContext<?> field.
context.annotation.illegalinject=RuntimeContext "{0}" have already been injected.
context.annotation.illegalfield=RuntimeContext field "{0}#{1}" is final and null, it can''t be injected.
context.annotation.illegalconfig.simple=Illegal configuration definition in annotation @Context(value="{0}", configurations=''{''..,"{1}",..''}''). The configuration "{1}" may not have been registered ?

context.parameter.empty=Parameter "{0}" have to be set for RuntimeContext "{1}"
context.parameter.causes.error=Parameter "{0}={1}" in RuntimeContext "{2}" causes error: "{3}"
context.parameter.illegal.use=Illegal use of parameter "{0}" in RuntimeContext "{1}", reason: "{2}"

# introspection error
context.provider.error.NoSuchConstructorException=Class "{0}" does not define a constructor like: {0}({1})
context.provider.error.InstantiationException=Class "{0}" can''t be instantiate due to underlying error: "{1}"
context.provider.error.IllegalAccessException=Class "{0}" does not define a accessible constructor like: {0}({1})
context.provider.error.InvocationTargetException=Error during constructor call "{0}({1})" - please check the stack trace following.

context.provider.error.default=Error has occurred due to nested exception.